TF2 Rich Presence/locales/Spanish.json

```diff
@@ -19,21 +19,21 @@
     "5581058060": "Ataque/Defensa",
     "2570455563": "Ataque/Defensa (Modo Medieval)",
     "5765869040": "El resultado final: ",
-    "1750272163": "No se puede conectar a Discord",
+    "1750272163": "No se puede conectar con Discord",
     "8452848160": "Capturar la Bandera",
     "7511751280": "Cambiar la configuración",
-    "7995884500": "El cambio de idioma requiere un reinicio para que surta efecto.",
-    "8011814710": "Compruebe si hay actualizaciones del programa al iniciar",
-    "9210445720": "Comprobar las actualizaciones",
+    "7995884500": "Cambiar el idioma requiere un reinicio para que surta efecto.",
+    "8011814710": "Comprobar si hay actualizaciones al iniciar",
+    "9210445720": "Buscar actualizaciones",
     "9227519100": "Clase",
     "3231589050": "Clase: {0}",
-    "1351878611": "Cerrar sin grabar",
+    "1351878611": "Cerrar sin guardar",
     "5860897400": "Punto de Control",
-    "1795231764": "No pude encontrar una instalación de Team Fortress 2.",
+    "1795231764": "No se pudo encontrar una instalación de Team Fortress 2.",
     "3315339845": "No se pudieron cargar las configuraciones, volviendo a los valores por defecto.{0}",
-    "1791761380": "Accidentes",
+    "1791761380": "Crasheos",
```

Comment:
"Crasheo" is an anglicism. "Accidentes" does not sound great either, but maybe something like "Fallos" sounds better and is more standard?